### PR TITLE
Features/array of selected places

### PR DESCRIPTION
**Pull request by gilemos**
Hey everyone!
So, I fixed the issue so that every page knows which places are selected and send the selected places to the schedule view controller. And, as you can see, the button to generate schedule in the more details view controller is inactive 

Video Walkthrough

### CODE REVIEW

**Comment by gilemos** [comment]
My gif is not working again aaaaa
Any ideas on how can I fix it?

**Comment by frankboamps** [comment]
> My gif is not working again aaaaa
> Any ideas on how can I fix it?

You can try checking the gif folder directory and making sure it's in the location your code expects it to be in.

**Comment by gilemos** [comment]
> > My gif is not working again aaaaa
> > Any ideas on how can I fix it?
> 
> You can try checking the gif folder directory and making sure it's in the location your code expects it to be in.

oh no, I do not mean the llamas, I mean the gif of the screen of the app for the pull request

**Comment by frankboamps** [comment]
> > > My gif is not working again aaaaa
> > > Any ideas on how can I fix it?
> > 
> > 
> > You can try checking the gif folder directory and making sure it's in the location your code expects it to be in.
> 
> oh no, I do not mean the llamas, I mean the gif of the screen of the app for the pull request

Video Walkthrough

**Comment by gilemos** [comment]
update: there was an issue with the schedule not updating when you selected other places, but not it is fixed:

Video Walkthrough

File: Travel Scheduler/View Controllers/HomeCollectionViewController.m [hunk 1]
```diff
@@ -17,10 +17,11 @@
 #import "APITesting.h"
 #import "PlaceObjectTesting.h"
 #import "SlideMenuUIView.h"
+#import "ScheduleViewController.h"
 @import GoogleMaps;
 @import GooglePlaces;
 
-@interface HomeCollectionViewController () <UITableViewDelegate, UITableViewDataSource, PlacesToVisitTableViewCellDelegate>
+@interface HomeCollectionViewController () <UITableViewDelegate, UITableViewDataSource, PlacesToVisitTableViewCellDelegate, DetailsViewControllerSetSelectedProtocol, PlacesToVisitTableViewCellSetSelectedProtocol, MoreOptionViewControllerSetSelectedProtocol>
```
Comment (aliu917) [inline Travel Scheduler/View Controllers/HomeCollectionViewController.m:25]: I think it they should all be delegates, not protocols?

**Review by aliu917 — APPROVED** [review]
Looks good! Great work!

**Pull request merged**